### ISSUE TITLE
Response Size limited?

### ISSUE THREAD

**Issue by cloud4every1** [marcoschwartz/aREST#263]
Hi all,

I try to retrieve some collected JSON data via a rest.variable on an MKR 1010 (ESP32 Wifi).
In general this works fine as long as the response does not exceed 197b.

When I convert the JSON and print it via the Serial, it contains all data. 
Is there a physical limit and/or a way to enlarge the response size?
I tried to overwrite
```
#undef MQTT_MAX_TRANSFER_SIZE
#define MQTT_MAX_TRANSFER_SIZE 1024
```
in my sketch, but this does not change any thing (if it would work at all).

many thanks for your help
Gerold ;-)

**Comment by OidaTiftla** [comment]
Hi Gerold,

it is maybe a silly question, but did you write the line `#define MQTT_MAX_TRANSFER_SIZE 1024` above the line `#include <aREST.h>`?

Like so:

```
#define MQTT_MAX_TRANSFER_SIZE 1024
#include <aREST.h>
...
```

**Comment by cloud4every1** [comment]
Hi,
No  `MQTT_MAX_TRANSFER_SIZE` was after `<aRest.h>` (I guess this is why it didn't work).
Haven't tried the `AREST_BUFFER_SIZE` but I will try both suggestions.

I adopted the` #define OUTPUT_BUFFER_SIZE` to 5000 and this helped at least to send now a bit more than 1000 characters.
As a workaround I limited the size of elements in the Json result array to 256 elements and request several times to get all. But this is not really satisfying.

Many thanks!
